lib/rbs_rails.rb

```diff
@@ -11,7 +11,8 @@
 module RbsRails
   class Error < StandardError; end
 
-  def self.copy_signatures(to:)
+  # @rbs to: untyped
+  def self.copy_signatures(to:) #: untyped
```

Comment:
I prefer `@rbs param: TYPE` style comment for parameters, and `#: TYPE` style comment for return value.
Please let me know if you have a favorite style.